### DISCUSSION TITLE
Grid forming inverter with PP

### DISCUSSION THREAD

**Discussion by SebastianCanargo** [e2nIEE/pandapower#2447] [Q&A]
Has anyone simulated a grid-forming inverter with Panda Power, maybe VSG (virtual synchronous generator)? I don't know how we can simulate it. 

Thanks for all the contributions.

**Comment by ccifrance** [comment]
I'm not quite sure if it is possible to simulate it with pander-power. I always simulate VSG with PowerFactory, EMTP-RV and PSCAD.

**Comment by vogt31337** [comment]
@SebastianCanargo , I've done something similar with a VSC while implementing them. You could create a small dc network with a dc source and a VSC in slack mode. This should be something kind of similar to a grid forming inverter. What exactly do you want to achieve?
